test/organizations/OrganizationCompletionEntity.js

```diff
@@ -0,0 +1,76 @@
+import { OrganizationEntity } from '../../src/organizations/OrganizationEntity.js';
```

Comment:
Good job coming up with your test cases!

Reply:
Testing is something I have done A LOT of in the past.